.github/workflows/ci.yaml

```diff
@@ -0,0 +1,133 @@
+        python-version: [ "3.6", "3.7", "3.8", "3.9" ]
+    env:
+      INVOKE_NAUTOBOT_PYTHON_VER: ${{ matrix.python-version }}
+      INVOKE_NAUTOBOT_LOCAL: "True"
+      NAUTOBOT_SELENIUM_URL: http://localhost:4444/wd/hub 
+      NAUTOBOT_SELENIUM_HOST: 172.18.0.1
+      NAUTOBOT_DB_PASSWORD: decinablesprewad
+      NAUTOBOT_DB_USER: nautobot
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres:13
+        # Provide the password for postgres
+        env:
+          POSTGRES_PASSWORD: decinablesprewad
+          POSTGRES_DB: nautobot
+          POSTGRES_USER: nautobot
+        ports:
+          - 5432:5432
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+      redis:
+        image: redis:6-alpine
+        ports:
+          - 6379:6379
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+      selenium:
+        image: selenium/standalone-firefox:latest
+        ports:
+          - 4444:4444
+    # The type of runner that the job will run on
+    runs-on: "${{ matrix.os }}"
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: "actions/checkout@v2"
+      # ----------------------------------------------
+      # Setup Python
+      # ----------------------------------------------
+      - uses: "actions/setup-python@v2"
+        with:
+          python-version: ${{ matrix.python-version }}
+      # ----------------------------------------------
+      # install & configure poetry
+      # ----------------------------------------------
+      - name: "Install Poetry"
+        uses: "snok/install-poetry@v1.1.6"
+        with:
+          virtualenvs-create: true
+          virtualenvs-in-project: true
+      # ----------------------------------------------
+      # load cached venv if cache exists
+      # ----------------------------------------------
+      - name: "Load cached venv"
+        id: "cached-poetry-dependencies"
+        uses: "actions/cache@v2"
+        with:
+          path: ".venv"
+          key: "venv-${{ runner.os }}--${{ matrix.python-version }}-${{ hashFiles('**/poetry.lock') }}"
+      # ----------------------------------------------
+      # install dependencies if cache does not exist
+      # ----------------------------------------------
+      - name: "Install dependencies"
+        if: "steps.cached-poetry-dependencies.outputs.cache-hit != 'true'"
+        run: "poetry install --no-interaction --no-root"
+      - name: Install Nautobot
+        run: poetry install --no-interaction
+      - name: Check migrations
+        if: matrix.python-version == '3.6'
+        run: poetry run invoke check-migrations
```

Comment:
Why restrict this to 3.6? Is it especially time-consuming?

Reply:
Trying to eliminate time wherever possible, if it's something we want we can add to other versions as well